docs/topics/serialization.txt

```diff
@@ -148,6 +148,12 @@ argument is passed in as ``True``::
 
     serializers.deserialize("xml", data, ignorenonexistent=True)
 
+In addition, you can extend the ``Deserializer`` class and extend it::
```

Comment:
I know this may be lame but waiting for your comments. The same for the release note.

Reply:
I think maybe we should mention how folks can add their own serialization format, so for example we should mention the `SERIALIZATION_MODULES` setting and how they can extend extisting `Deserializer`s

I think this should be within the `Serialization formats` section, maybe at the bottom after `YAML`, as `Custom  serialization formats` (or something). This can have a `.. _custom-serialization-formats:` reference link we can use for the release notes.

It will also need a `.. versionchanged: 5.2` note

Reply:
I have tried to rewrite the docs a bit, can you review the updates? Would be good to get more eyes on them for sure